src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -358,14 +361,19 @@ def get_beacon_config(
     recipe = GetHttpRequestRecipe(
         endpoint="/eth/v1/node/identity", port_id=BEACON_HTTP_PORT_ID
     )
+    files = {
+        constants.GENESIS_DATA_MOUNTPOINT_ON_CLIENTS: el_cl_genesis_data.files_artifact_uuid
+    }
 
+    if persistent:
+        files[BEACON_DATA_DIRPATH_ON_BEACON_SERVICE_CONTAINER] = Directory(
+            persistent_key="beacon_data_folder",
```

Comment:
this shall be replaced with something unique, similar for other persistent_keys, so that the name doesn't clash with service name